welltestpy/data/testslib.py

```diff
@@ -467,6 +467,26 @@ def plot(self, wells, exclude=None, fig=None, ax=None, **kwargs):
             **kwargs
         )
 
+    def diagnostic_plot(self, wells, **kwargs):
+        """Make a diagnostic plot.
+
+
+        Parameters
+        ----------
+       wells : :class:`dict`
+            Dictionary containing the well classes sorted by name.
+
+        Notes
+        -----
+        This will be used by the Campaign class.
+        """
+        observation = self.observations[wells]
```

Comment:
Please check if the given observation_well is actually present in `self.observations`.